README.md

```diff
@@ -46,11 +46,36 @@ Other devices may be compatible and all that is needed is to add the ID to apex-
 
 ## Installation
 
+For installing this software, follow the steps:
+
 ### UDev
 
-Enter the following data from [here](https://gist.github.com/ToadKing/d26f8f046a3b707e9e4b9821be5c9efc) (Shoutout to @ToadKing](https://github.com/ToadKing).
+1. Get the device id: `lsusb | grep "SteelSeries Apex"`:
+
+```shell
+$ lsusb | grep "SteelSeries Apex"
+Bus 001 Device 002: ID 1038:1610 SteelSeries ApS SteelSeries Apex Pro
+```
+
+The **id** is the left part of the ID.
```

Comment:
the product id that we want is actually the right part, the left side is the brand/vendor(/steelseries)'s ID, with that ID it can be a mouse, a keyboard a headset, however the right part is the product ID